e2e/cypress/integration/find.test.js

```diff
@@ -0,0 +1,134 @@
+context('Find', () => {
+  beforeEach(() => {
+  });
+
+  const login = () => {
+    cy.get('[data-cy=login-form-uri]').clear().type('http://smaug.frontend-staging.svc.cloud.dbc.dk:8001');
+    cy.get('[data-cy=login-form-username]').clear().type('admin');
+    cy.get('[data-cy=login-form-password]').clear().type('password_stg');
```

Comment:
We can't have login info lying around on github. Kasper has created a docker setup that works around this issue. Can you use that instead?